app/controllers/pipeline_outputs_controller.rb

```diff
@@ -1,7 +1,7 @@
 class PipelineOutputsController < ApplicationController
   include ReportHelper
   include PipelineOutputsHelper
-  before_action :set_pipeline_output, only: [:show, :show_taxid_fasta, :send_nonhost_fasta, :send_unidentified_fasta]
+  before_action :set_pipeline_output, only: [:show, :show_taxid_fasta, :send_nonhost_fasta, :send_unidentified_fasta, :show_alignment_info]
```

Comment:
oops. pipeline_outputs_controller is gone.

Reply:
haha